PowerEditor/installer/nativeLang/Malayalam.xml

```diff
@@ -0,0 +1,954 @@
+                    <Item id="24473" name="സ്ടിലെർ"/>
+                    <Item id="24501" name="ഡെലിമീറ്ററിന്റെ 5 രീതിയിൽ"/>
+                    <Item id="24520" name="തുറക്കുക:"/>
+                    <Item id="24521" name="എസ്കേപ്പ്:"/>
+                    <Item id="24522" name="അടയ്ക്കുക:"/>
+                    <Item id="24523" name="സ്ടിലെർ"/>
+                    <Item id="24551" name="ഡെലിമീറ്ററിന്റെ 6 രീതിയിൽ"/>
+                    <Item id="24570" name="തുറക്കുക:"/>
+                    <Item id="24571" name="എസ്കേപ്പ്:"/>
+                    <Item id="24572" name="അടയ്ക്കുക:"/>
+                    <Item id="24573" name="സ്ടിലെർ"/>
+                    <Item id="24601" name="ഡെലിമീറ്ററിന്റെ 7 രീതിയിൽ"/>
+                    <Item id="24620" name="തുറക്കുക:"/>
+                    <Item id="24621" name="എസ്കേപ്പ്:"/>
+                    <Item id="24622" name="അടയ്ക്കുക:"/>
+                    <Item id="24623" name="സ്ടിലെർ"/>
+                    <Item id="24651" name="ഡെലിമീറ്ററിന്റെ 8 രീതിയിൽ"/>
+                    <Item id="24670" name="തുറക്കുക:"/>
+                    <Item id="24671" name="എസ്കേപ്പ്:"/>
+                    <Item id="24672" name="അടയ്ക്കുക:"/>
+                    <Item id="24673" name="സ്ടിലെർ"/>
+                </Operator>
+            </UserDefine>
+            <Preference title="Preferences">
+                <Item id="6001" name="Closeഅടയ്ക്കുക"/>
+                <Global title="General">
+                    <Item id="6101" name="ടൂൾബാർ"/>
+                    <Item id="6102" name="മറയ്ക്കുക"/>
+                    <Item id="6103" name="ചെറിയചിഹ്നങ്ങള്"/>
+                    <Item id="6104" name="ബിഗ് ഐക്കണുകൾ"/>
+                    <Item id="6105" name="ബിഗ് ഐക്കണുകൾ..."/>
+
+                    <Item id="6106" name="ടാബ് ബാർ"/>
+                    <Item id="6107" name="കുറയ്ക്കുക"/>
+                    <Item id="6108" name="ലോക്ക് (യാതൊരു വലിച്ചിടുന്നത്)"/>
+                    <Item id="6109" name="നിഷ്ക്രിയ ടാബുകളുടെ ഇരുണ്ട്"/>
+                    <Item id="6110" name="സജീവ ടാബിൽ നിറമുള്ള ബാർ വരയ്ക്കുക"/>
+
+                    <Item id="6111" name="സ്റ്റാറ്റസ് ബാർ കാണിക്കുക"/>
+                    <Item id="6112" name="ഓരോ ടാബിൽ അടയ്ക്കൂ ബട്ടൺ കാണിക്കുക"/>
+                    <Item id="6113" name="ഇരട്ട പ്രമാണം അടയ്ക്കാൻ ക്ലിക്ക്"/>
+                    <Item id="6118" name="മറയ്ക്കുക"/>
+                    <Item id="6119" name="മൾട്ടി-ലൈൻ"/>
+                    <Item id="6120" name="ലംബമായ"/>
+
+                    <Item id="6122" name="മറയ്ക്കുക (ടോഗിൾ Alt അല്ലെങ്കിൽ F10 കീ ഉപയോഗിക്കുക)"/>
+                    <Item id="6123" name="ലോക്കലൈസേഷൻ"/>
+
+                    <Item id="6125" name="ഡോക്യുമെന്റ് പട്ടിക പാനൽ"/>
+                    <Item id="6126" name="കാണിക്കുക"/>
+                    <Item id="6127" name="വിപുലീകരണം കോളം അപ്രാപ്തമാക്കുക"/>
+                </Global>
+                <Scintillas title="Editing">

```

Comment:
@roshan0 I think you need to translate this too and similarly others as well. Like "Print", "Backup" etc. which as used in title